packages/core/src/lib/filter/index.ts

```diff
@@ -14,13 +14,13 @@ import type {
   ProtocolCreateOptions,
   ProtocolOptions,
 } from "@waku/interfaces";
+import { WakuMessage as WakuMessageProto } from "@waku/proto";
 import {
   getPeersForProtocol,
   selectConnection,
   selectPeerForProtocol,
   selectRandomPeer,
-} from "@waku/libp2p-utils";
-import { WakuMessage as WakuMessageProto } from "@waku/proto";
+} from "@waku/utils";
```

Comment:
`from @waku/utils/libp2p` is the expectation from the original issue.

Reply:
Cc @danisharora099 this PR does not implement the behaviour described in the issue it closes.